apps/modernization-ui/src/apps/page-builder/pages/ValuesetLibrary/FilterModal/Filter.tsx

```diff
@@ -0,0 +1,189 @@
+import { Button, Icon, Tag } from '@trussworks/react-uswds';
+import React, { useContext, useEffect, useState } from 'react';
+import { SelectControl } from 'components/FormInputs/SelectControl';
+import { useForm } from 'react-hook-form';
+import { MultiSelectInput } from 'components/selection/multi';
+import { fetchConditions } from '../../../services/conditionAPI';
+import { Condition } from '../../../generated';
+import { UserContext } from '../../../../../providers/UserContext';
+import { FilterWrapper } from '../../FilterModal/FilterWrapper';
+import { FilterPanel } from '../../FilterModal/FilterPanel';
+import {
+    statusOptions,
+    eventYpeOption,
+    valueSetFieldList,
+    arithOperator,
+    dateOperator,
+    initOperator
+} from '../../../constant/constant';
+import { ValueSetsContext } from '../../../context/ValueSetContext';
+
+export const FilterButton = () => {
```

Comment:
Your file names and component names are not consistent and are very confusing. Is this a filter modal or a filter button?  is it for the valueset or the question library.  File names and react function names should be the same for example.

QuestionLibrary -> FilterModal -> QuestionLibraryFilterModal.tsx, and the component name should be QuestionLibraryFilterModal